src/ci.js

```diff
@@ -116,6 +116,21 @@ exports.getVars = function() {
     }
     return result;
   }
+  // Github actions
+  if(env.CI === 'true' && env.GITHUB_ACTIONS === 'true'){
+
+    var branchName = env.GITHUB_REF || '';
```

Comment:
sorry @luisVelasquezSauce - can you not default to empty string? better to leave it undefined, and it will get replaced with a default branch name

instead suggest updating condition below:

```
if (branchName && branchName.indexOf(....
```

Reply:
Ok, updated